apps/pie-docs/src/patterns/cookie-banner/code/events.json

```
@@ -0,0 +1,29 @@
+{
+  "headings": [
+    "Event",
+    "Type",
+    "Description"
+  ],
+  "rows": [
+    [
+      "`pie-cookie-banner-accept-all`",
+      "`CustomEvent`",
+      "Triggered when `accept all` is clicked."
+    ],
+    [
+      "`pie-cookie-banner-necessary-only`",
+      "`CustomEvent`",
+      "Triggered when `necessary` is clicked."
+    ],
+    [
+      "`pie-cookie-banner-manage-prefs`",
+      "`CustomEvent`",
+      "Triggered when `manage prefs` is clicked."
+    ],
+    [
+      "`pie-cookie-banner-prefs-saved`",
+      "`CustomEvent`",
+      "Triggered when `manage prefs` saved is clicked."
```

I think being less specific here can actually be a little clearer.
```suggestion
      "Triggered when the user chooses to accept all cookies."
    ],
    [
      "`pie-cookie-banner-necessary-only`",
      "`CustomEvent`",
      "Triggered when the user chooses to accept necessary cookies only."
    ],
    [
      "`pie-cookie-banner-manage-prefs`",
      "`CustomEvent`",
      "Triggered when the user chooses to manage their cookie preferences."
    ],
    [
      "`pie-cookie-banner-prefs-saved`",
      "`CustomEvent`",
      "Triggered when the user saves their cookie preferences."
```